backend/agents/grading_agent.py

```
@@ -65,23 +65,28 @@ def _create_grading_tools(self) -> List[BaseTool]:
-        def analyze_user_responses(user_responses: str, success_metric: str) -> str:
-            """Analyze user responses against success metrics"""
-            return f"Analyzing {len(user_responses.split())} words of user responses against success metric: {success_metric}"
+        def analyze_business_thinking(responses: str, success_metric: str) -> str:
+            """Analyze user responses for business thinking quality and strategic analysis"""
+            return f"Analyzing business thinking in {len(responses.split())} words against success metric: {success_metric}. Evaluating strategic perspective, problem identification, and solution development."
         
         @tool
-        def evaluate_learning_objectives(responses: str, objectives: str) -> str:
-            """Evaluate responses against learning objectives"""
-            return f"Evaluating responses against {len(objectives.split(','))} learning objectives"
+        def evaluate_strategic_depth(responses: str, objectives: str) -> str:
```

Ruff (0.13.1)

Unused function argument: `responses`

(ARG001)